statusHandler/interface.go

```diff
@@ -0,0 +1,39 @@
+package statusHandler
```

Comment:
might move this file in statusHandler/view package

Reply:
Done.